src/native/minipal/random.c

```diff
@@ -86,6 +95,9 @@ int32_t minipal_get_cryptographically_secure_random_bytes(uint8_t* buffer, int32
     {
         return 0;
     }
+#elif HAVE_BCRYPT_H
+    NTSTATUS status = BCryptGenRandom(NULL, buffer, (ULONG)bufferLength, BCRYPT_USE_SYSTEM_PREFERRED_RNG);
+    return BCRYPT_SUCCESS(status) ? 1 : 0;
```

Comment:
Are these return codes correct? Looking at other returns, it seems like `0` is success and `-1` is failure.